client/client-multi/client-c/src/kaa/kaa_logging.c

```diff
@@ -34,45 +34,44 @@
 #include "utilities/kaa_log.h"
 #include "avro_src/avro/io.h"
 
-
-
 #define KAA_LOGGING_RECEIVE_UPDATES_FLAG   0x01
 #define KAA_MAX_PADDING_LENGTH             (KAA_ALIGNMENT - 1)
-
-
+#define KAA_MAX_LOGS_IN_BUCKET             64
+#define KAA_MAX_BUCKET_SIZE                1024
```

Comment:
Please, check whether KAA_MAX_BUCKET_SIZE doesn't exceed the size of a OUT transport buffer.
See defaults.h